cli/commands/application.go

```diff
@@ -91,10 +87,16 @@ func (cmd *Application) show(appName string) {
 	}
 	table.Add("Type:", application.Spec.Type)
 	table.Add("CatalogItemId:", entity.CatalogItemId)
-	table.Add("LocationId:", strings.Join(application.Spec.Locations, ", "))
-	table.Add("LocationName:", location.Name)
-	table.Add("LocationSpec:", location.Spec)
-	table.Add("LocationType:", location.Type)
+	if len(application.Spec.Locations) > 0 {
```

Comment:
looks good.  

Might be worth iterating over the locations if there are multiple ones, something like
```
for _,  loc := range application.Spec.Locations {
	table.Add("LocationId:", loc.Id)
	table.Add("LocationName:", loc.Name)
	table.Add("LocationSpec:", loc.Spec)
	table.Add("LocationType:", loc.Type)
}
```
Caveat: haven't tried this out.